lib/normalize.js

```diff
@@ -32,6 +33,16 @@ var _splitLocator = function (nemo, locatorString) {
 module.exports = function normalize(nemo, _locator) {
   var locator = _locator;
   var normalizedLocator;
+
+  if (locator.using && locator.value && nemo.wd.By[locator.using]) {
+    normalizedLocator = nemo.wd.By[locator.using](locator.value);
+    if(normalizedLocator.constructor === locator.constructor){
```

Comment:
Yes, returning `normalizedLocator` would be logically correct and simpler.  I did the check in order to explicitly return the input object, so as to show from the function caller's perspective that no normalization was necessary.  Ideally, the `nemo.wd.By` call on line 38 wouldn't be necessary to check if the input is already normalized, and this type of conditional would make more sense, but I ended up with this since I couldn't find easy access to `webdriver.Locator` constructor function.